.storybook/main.js

```diff
@@ -24,13 +28,21 @@ module.exports = {
     },
     "@storybook/addon-controls",
   ],
-  // We want the repo's webpack configuration to be a part of the
-  // Storybook configuration for SCSS and SVGs.
-  webpackFinal: config => ({
-    ...config,
-    module: {
-      ...config.module,
-      rules: custom.module.rules,
-    },
-  }),
+  // We want the repo's webpack configuration to be a part of the Storybook
+  // configuration for SCSS and SVGs, but we want to keep the existing
+  // configuration loaders to use MDX when the project builds.
+  webpackFinal: config => {
+    const { rules } = config.module;
+    // We still want MDX-related rules. This is not ideal as the order of the
```

Comment:
Nit:  I think it'll be more useful if this comment explicitly says
1) that rules[4] copies over the mdx specific rule, and will break if the mdx rule is moved
2) that custom webpack rules are mostly to handle SVG loading and that we may be able to use the default config if we fix SVG loading.  

I've seen so many webpack configs that I've been afraid to touch for fear of breaking it, and hopefully making it explicit why it is this way will give future people confidence that they can change it.